easypost/rate.py

```diff
@@ -0,0 +1,5 @@
+from easypost.resource import CreateResource
+
+
+class Rate(CreateResource):
```

Comment:
This should probably only inherit `Resource`

Reply:
Nice catch, looks like we never fix this in the past